client/app/nonComp/components/ReportPage/Conditions/DecisionReviewType.jsx

```diff
@@ -1,5 +1,37 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import Checkbox from 'app/components/Checkbox';
 
-export const DecisionReviewType = () => {
-  return <div>decision review type test component please ignore</div>;
+const CHECKBOX_OPTIONS = [
+  {
+    label: 'Higher-Level Reviews',
+    name: 'HigherLevelReviews'
```

Comment:
If you want it to match what the database will be expecting then it is singularized. We could always parse it out later if we needed to, but I figure it's easy enough to change here.
```suggestion
    name: 'HigherLevelReview'
```